fellow-submissions/stephanie-smith/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link href="https://fonts.googleapis.com/css?family=Amatic+SC&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/blocks.css/dist/blocks.min.css" />
+    <title>Quiz Time!</title>
+  </head>
+  
+  <body>
+    <h1>Music Theory Quiz!</h1>
+    <p>Hello, and welcome to my quiz! The topic of this quiz is music theory, but it's alright if you don't know much. Hopefully you learn
+    something from it! Enjoy!</p>
+    
+    <form action="test" method="post">
+      <fieldset class="block fixed color themed">
+        <legend>Question #1</legend>
+        <label for="First Question">Let's start easy. <strong>What note lasts 2 beats?</strong></label>
+        <ul>
+          <li><input type="radio" name="note" value="whole">Whole note</li>
+          <li><input type="radio" name="note" value="quarter">Quarter note</li>
+          <li><input type="radio" name="note" value="half">Half note</li>
+          <li><input type="radio" name="note" value="eighth">Eighth note</li>
+          </ul>
+      </fieldset>
```

Comment:
Each `<input>` needs a label, not the question for the section. You can simply use a `<p>` tag to label the question. Very important though that each radio button has an associated label.